src/equations/shallow_water_two_layer_1d.jl

```diff
@@ -0,0 +1,594 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+# TODO: Update variables in docstring
+# TODO: Check Variables
```

Comment:
Still to do?

Reply:
The variables have been checked, I removed the comments in commit 4fd850aa42033387973d2b5a9338294706e416cc